### PR TITLE
mtkennerly.shawl version 1.5.0

### PR DESCRIPTION
**Pull request by mtkennerly**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/143761)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Packages\mtkennerly.shawl__DefaultSource\shawl.exe returned exit code: 2
> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\shawl.exe returned exit code: 2
> Executable C:\Users\Validator\AppData\Local\Microsoft\WinGet\Packages\mtkennerly.shawl__DefaultSource\shawl.exe returned exit code: 2
> Executable C:\Users\Validator\AppData\Local\Microsoft\WinGet\Links\shawl.exe returned exit code: 2

(Automated response - build 869.)

**Comment by mtkennerly** [comment]
@stephengillie The executable requires arguments. Is there a way to instruct the validation to run `shawl --help` instead of just `shawl`?

File: manifests/m/mtkennerly/shawl/1.5.0/mtkennerly.shawl.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: mtkennerly.shawl
+PackageVersion: 1.5.0
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: shawl.exe
+  PortableCommandAlias: shawl
+Installers:
```
Comment (stephengillie) [inline manifests/m/mtkennerly/shawl/1.5.0/mtkennerly.shawl.installer.yaml:11]:
```suggestion
Commands: --help
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @mtkennerly,

~~The [`Commands` parameter](https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.6.0/installer.md) seems to be needed here. This feature doesn't get used often, so I haven't seen many examples.~~

File: manifests/m/mtkennerly/shawl/1.5.0/mtkennerly.shawl.installer.yaml (hunk 1 above)
Comment (mtkennerly) [reply manifests/m/mtkennerly/shawl/1.5.0/mtkennerly.shawl.installer.yaml:13]: Thanks! I've just pushed the update (had to use an array).

**Comment by mdanish-kh** [comment]
> @stephengillie The executable requires arguments. Is there a way to instruct the validation to run `shawl.exe --help` instead of just `shawl.exe`?

@stephengillie - I don't think the commands parameter can be used for this purpose. To my knowledge, the purpose of commands fields is:
- Allows the package to be shown in a filter like `winget search --command <cmd>` e.g, `winget search --command nuget`. PowerToys takes advantage of this search filter in its Command Not Found module.
- For portables (non-zip), this field is used to rename the executable in non-symlinked installations and in symlinked installations, the symlink is created with this name.
- For nested portables (zip - portable), the function above is carried out with the `PortableCommandAlias` field so I believe `Commands:` field currently just serves to show those packages under `winget search --comand` filter

`Commands: --help` doesn't seem to be the correct value to put here, it should be `shawl` i.e., the root command someone can use to invoke this package.

If we want user to use `shawl --help` post-installation, we can mention this under the `InstallationNotes` field in the default locale.
Something like 

```
InstallationLocale: |-
  Run "shawl --help" to view all the supported arguments and sub-commands
```

File: manifests/m/mtkennerly/shawl/1.5.0/mtkennerly.shawl.installer.yaml (hunk 1 above)
Comment (mtkennerly) [reply manifests/m/mtkennerly/shawl/1.5.0/mtkennerly.shawl.installer.yaml:11]: I've reverted this change based on https://github.com/microsoft/winget-pkgs/pull/143761#issuecomment-1989004315 .

**Review by mdanish-kh — APPROVED** [review]

**Comment by mdanish-kh** [comment]
I believe this requires a waiver

**Review by stephengillie — APPROVED** [review]

**Comment by stephengillie** [comment]
@mtkennerly, sorry for giving bad advice.

**Pull request merged**